scripts/version-ldflags

```diff
@@ -19,7 +19,9 @@
 # version info into cmd/version.go via ldflags
 GIT_VERSION=$(git describe --tags --always --dirty)
 GIT_HASH=$(git rev-parse HEAD)
-BUILD_DATE=$(date +'%Y-%m-%dT%H:%M:%SZ')
+# https://reproducible-builds.org/docs/source-date-epoch/
+# https://mirrors.edge.kernel.org/pub/software/scm/git/docs/git-log.html
+SOURCE_DATE_EPOCH=$(git log --date=iso8601-strict -1 --pretty=%ct)
```

Comment:
This may not matter, but I wonder if this date should only be set if the tree state is "clean", if the tree state is "dirty" we just set current time?

Reply:
Sorry, I mean to answer and hit the resolve button.

I agree this is a good suggestion. We could do this later.